module/platformUserRoles/helper.js

```diff
@@ -1,38 +1,9 @@
 const platformRolesHelper = require(ROOT_PATH + "/module/platformRoles/helper")
+let sunBirdService =
```

Comment:
@deepakr123 , @rakeshSgr - This helper is not used but still imported

Reply:
resolved